Core/Debugger/DisassemblyManager.cpp

```diff
@@ -396,7 +396,7 @@ u32 DisassemblyManager::getNthNextAddress(u32 address, int n)
 		analyze(address);
```

Comment:
should we avoid analyzing if the address has gotten unaligned? It feels like there are some early checks possible, rather than just aligning the return value?

Reply:
Well, that actually aligns the address, and I didn't want to change that logic.

-[Unknown]